x-pack/test/functional_with_es_ssl/apps/cases/create_case_form.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import uuid from 'uuid';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObject, getService }: FtrProviderContext) => {
+  describe('Create case ', function () {
+    const common = getPageObject('common');
+    const find = getService('find');
+    const casesApp = getService('casesApp');
+    const casesAppApi = getService('casesAppApi');
+
+    before(async () => {
+      await common.navigateToApp('casesStackManagement');
+    });
+
+    after(async () => {
+      casesAppApi.deleteAllCases();
+    });
+
+    describe('creating a case', () => {
```

Comment:
This sub-suite could be removed as we're already in the `Create case` suite and the edit tests are now in a separate suite.